platform/web/js/libs/library_godot_audio.js

```diff
@@ -588,18 +615,55 @@ class SampleNode {
 		return this._sampleNodeBuses.get(bus);
 	}
 
+	/**
+	 * Sets up and assigns AudioWorkletProcessor used to track playback position.
+	 * @param {boolean} Whether or not attempt to add the worklet module if it hasn't been.
+	 * @returns {AudioWorkletProcessor}
+	 */
+	async createPositionWorklet(retry) {
+		let worklet;
+		try {
+			worklet = new AudioWorkletNode(
+				GodotAudio.ctx,
+				'godot-position-reporting-processor'
+			);
+			worklet.port.onmessage = (event) => {
+				switch (event.data['type']) {
+				case 'position':
+					this._playbackPosition = (parseInt(event.data.data, 10) / this.getSample().sampleRate) + this.offset;
+					break;
+				default:
+				// do nothing
+				}
+			};
+		} catch (e) {
+			if (retry) {
+				const path = GodotConfig.locate_file('godot.audio.worklet.js');
```

Comment:
We need to make another file, as the current worklet is made for threaded audio. I suggest `godot.audio.position.worklet.js`.